src/book.py

```diff
@@ -363,6 +363,123 @@ def _read_kraken_ledgers_old(self, file_path: Path) -> None:
 
         self._read_kraken_ledgers(file_path)
 
+    def _read_bitpanda_pro_trades(self, file_path: Path) -> None:
+        """
+        Read trade statement from Bitpanda Pro.
```

Comment:
Add Args to docstring. And title in the first line. 
 Checkout google docstring guide for further informations or have a look at some other docstrings.